.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Nice! You found an issue! :) Get help faster by providing the following details:
```

Comment:
About "code to demonstrate it", what if it's an improvement or new feature? Isn't it enough to have the block of code at the end to imply this?